cookbooks/aws-parallelcluster-environment/files/login_nodes/keys-manager.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+set -e
+#
+# Manage the creation and usage of the keys in the login nodes
+#
+# --create                      Create the required keys in the specified folder
+# --import                      Copy the keys from the specified folder to the ssh folder
+# --folder_keys <folder_path>   Full path of the folder which contains the generated keys
+# --help                        Print this help message
+
+function error() {
+  >&2 echo "[ERROR] $1"
+  exit 1
+}
+
+function info() {
+  echo "[INFO] $1"
+}
+
+function help() {
+  local -- cmd=$(basename "$0")
+  cat <<EOF
+
+  Usage: ${cmd} [OPTION]...
+
+  Manage the creation and usage of the keys in the login nodes
+
+  --create                      Create the required keys in the specified folder
+  --import                      Copy the keys from the specified folder to the ssh folder
+  --folder-path <folder_path>   Full path of the folder which contains the generated keys
+  --help                        Print this help message
+EOF
+}
+
+function create_keys() {
+  ssh-keygen -t ecdsa -f "$FOLDER_PATH/ssh_host_ecdsa_key" -q -P ""
+  ssh-keygen -t ed25519 -f "$FOLDER_PATH/ssh_host_ed25519_key" -q -P ""
+  if is_not_alinux; then
+      ssh-keygen -t rsa -f "$FOLDER_PATH/ssh_host_rsa_key" -q -P ""
+  fi
+  if is_ubuntu; then
+    ssh-keygen -t dsa -f "$FOLDER_PATH/ssh_host_dsa_key" -q -P ""
+  fi
+}
+
+function import_keys() {
+  rm -f /etc/ssh/ssh_host_*
```

Comment:
I'd add a `echo "Importing ssh host keys"` to help trace the execution in case of debugging...